components/Header/index.js

```diff
@@ -9,4 +9,31 @@
 //  </div >
 // And add it to the DOM in the .header-container component
 
-function Header() {}
+
+
+function header() {
+    const 
+        headerCont = document.querySelector('.header-container'),
```

Comment:
If this is the selection of "where you want something to go" should it be inside the function that is creating that "something"?